packages/react-native/Libraries/StyleSheet/processTransform.js

```diff
@@ -256,6 +260,14 @@ function _validateTransform(
       break;
     case 'translateX':
     case 'translateY':
+      invariant(
+        typeof value === 'number' ||
+          (typeof value === 'string' && value.endsWith('%')),
+        'Transform with key of "%s" must be number or a percentage. Passed value: %s.',
+        key,
+        stringifySafe(transformation),
+      );
```

Comment:
It looks like this is an existing pattern, but for the future, we've been trying to move towards no-oping on invalid values, instead of creating a redbox, or something more disruptive.

Even longer into the future (not worth thinking about yet), on Fabric side, we want to just do all this parsing in native.

Reply:
All of this parsing does happen in native. `validateTransform` is just an additional DEV-time only validation of the input to get better signal to the developer.

Reply:
Yep, but we have though redbox during development on invalid style is too aggressive/disruptive.